mediapipe/tasks/ios/components/processors/utils/sources/MPPClassifierOptions+Helpers.mm

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "mediapipe/tasks/ios/common/utils/sources/NSString+Helpers.h"
+#import "mediapipe/tasks/ios/components/processors/utils/sources/MPPClassifierOptions+Helpers.h"
+
+namespace {
+using ClassifierOptionsProto = ::mediapipe::tasks::components::processors::proto::ClassifierOptions;
+}
+
+@implementation MPPClassifierOptions (Helpers)
+- (void)copyToProto:(ClassifierOptionsProto *)classifierOptionsProto {
+  if (self.displayNamesLocale) {
```

Comment:
If you use this style of assignment, you probably want to clear the proto of all values first.